include/ginkgo/core/stop/residual_norm.hpp

```diff
@@ -121,8 +121,8 @@ class ResidualNormReduction : public ResidualNorm<ValueType> {
         /**
          * Factor by which the residual norm will be reduced
          */
-        remove_complex<ValueType> GKO_FACTORY_PARAMETER_SCALAR(reduction_factor,
-                                                               1e-15);
+        remove_complex<ValueType> GKO_FACTORY_PARAMETER_SCALAR(
+            reduction_factor, static_cast<remove_complex<ValueType>>(1e-15));
```

Comment:
It's only defined in `core/test/utils.hpp` right now, but should we use the `reduction_factor` from there for the default value here?

Reply:
Yes, but I would prefer to do that in another PR